### PR TITLE
added karmaStore dict

### PR DESCRIPTION
**Pull request by OliverTansley**

### CODE REVIEW

File: karmabot/karma/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from karmabot.karma.karma_store import KarmaStore
```
Comment (Joeyh021) [inline karmabot/karma/__init__.py]: ugly ass import, plz fix

File: karmabot/karma/__init__.py [hunk 2]
```diff
@@ -6,6 +7,8 @@
 
 async def setup(bot: Bot):
 
-    await bot.add_cog(AdminCommands(bot))
-    await bot.add_cog(UserCommands(bot))
-    await bot.add_cog(Listeners(bot))
+    karma_store = KarmaStore()
```
Comment (Joeyh021) [inline karmabot/karma/__init__.py:10]: nicely done here

File: karmabot/karma/admin_commands.py [hunk 3]
```diff
@@ -2,5 +2,6 @@
 
 
 class AdminCommands(commands.Cog):
-    def __init__(self, bot):
+    def __init__(self, bot,karma_store):
         self.bot = bot
+        AdminCommands.karma_store = karma_store
```
Comment (Joeyh021) [inline karmabot/karma/admin_commands.py]: this can just be `self.karma_store = karma_store`

File: karmabot/karma/user_commands.py [hunk 4]
```diff
@@ -1,6 +1,20 @@
 import discord.ext.commands as commands
 
+from karmabot import karma
+from karmabot.karma import karma_store
```
Comment (Joeyh021) [inline karmabot/karma/user_commands.py]: bro fix ur imports fr, run pylint and it should tell u how to clean them up

File: karmabot/karma/admin_commands.py [hunk 5]
```diff
@@ -2,5 +2,6 @@
```
Comment (Joeyh021) [inline karmabot/karma/admin_commands.py:2]: need an import here for karma store

File: karmabot/karma/__init__.py (hunk 2 above)
Comment (OliverTansley) [reply karmabot/karma/__init__.py:10]: thanks :D

**Comment by Joeyh021** [comment]
LGTM !

**Pull request merged**